### PR TITLE
ARO-9986 | Add azure-first-party-application-client-id flag to CS

### PR DESCRIPTION
**Pull request by cgiradkar**
### What this PR does

Jira: 
Link to demo recording:

### CODE REVIEW

**Comment by mjlshen** [comment]
@cgiradkar I've sent you an invite to this repo. To pass CI, you'll need to close and recreate this PR as a branch of `Azure/ARO-HCP` (not from a fork) - thanks!

**Comment by cgiradkar** [comment]
> @cgiradkar I've sent you an invite to this repo. To pass CI, you'll need to close and recreate this PR as a branch of `Azure/ARO-HCP` (not from a fork) - thanks!

will do. Thanks

**Comment by cgiradkar** [comment]
Closing PR and creating another without fork.

File: cluster-service/deploy/openshift-templates/arohcp-service-template.yml [hunk 1]
```diff
@@ -513,6 +518,7 @@ objects:
           - --cloud-region-config=/configs/cloud-resources/cloud-regions.yaml
           - --cloud-region-constraints-config=/configs/cloud-resource-constraints/cloud-region-constraints.yaml
           - --azure-auth-config-path=/secrets/azure-credentials/azure-auth-config
+          - --azure_fpa_client_id=${AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID}
```
Comment (JameelB) [inline cluster-service/deploy/openshift-templates/arohcp-service-template.yml:521]: The `azure_fpa_client_id` is only used for the makefile. This should be: 

```suggestion
          - --azure-first-party-application-client-id=${AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID}
```

**Pull request closed without merging**